README.md

```diff
@@ -1,98 +1,112 @@
-standard no longer requires it:
+This gem does not include the `ie8` version of GOV.UK Frontend, as the service
+standard no longer requires support for Internet Explorer versions less than 11:
 
 https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices
 
-This gem and its documentation cannot tell you how to use the GOVUK effectively,
-see the GOVUK Design System documentation for that:
+This gem and its documentation cannot tell you how to use the GOV.UK patterns
+effectively, see the GOV.UK Design System documentation for that:
 
-https://design-system.service.gov.uk/ 
+https://design-system.service.gov.uk/
 
 ## Installation
 
 Add this line to your application's Gemfile:
 
-`ruby gem 'dxw_govuk_frontend_rails'`
+```ruby
+gem 'dxw_govuk_frontend_rails'
+```
 
 And then execute:
 
-`bundle`
+```
+$ bundle
+```
+
+## Usage
+
+Your Rails app will need to have SCSS enabled.
 
-## Usage Your Rails app will need to have sass enabled.
+**IMPORTANT: You must not use this library without changes if the service you
+are building is not on GOV.UK. Read and understand the guidance here:**
+
+https://www.gov.uk/service-manual/design/making-your-service-look-like-govuk#if-your-service-isnt-on-govuk
 
 ### Basic usage
 
-If the service you are building does not need to modify or extend the GOVUK
-Frontend all you need to do is import the main sass and javascript files into
+If the service you are building does not need to modify or extend the GOV.UK
+Frontend, all you need to do is import the main SCSS and JavaScript files into
 your Rails app:
 
-- create a sass file in your application: 
+- create a SCSS file in your application:
 
-`app/assets/stylesheets/govuk_frontend_rails.scss`
-- import the styles into `govuk_frontend_rails.scss`: 
+  `app/assets/stylesheets/govuk_frontend_rails.scss`
 
-`@import "govuk-frontend-rails";`
-- require the `govuk-frontend-rails.scss` file in `application.css` or
-  equivalent: 
+- import the styles into `govuk_frontend_rails.scss`:
+
+  `@import "govuk-frontend-rails";`
+
+- import the `govuk-frontend-rails.scss` file in the comment block in
+  `app/assets/stylesheets/application.css` or equivalent:
 
   `*= require govuk_frontend_rails`
-- require the javascript into `app/assets/javascripts/application.js`: 
 
-`//= require govuk_frontend_rails`
-- initialise the GOVUK Frontend either in a .js file on in your application
+- import the JavaScript in the comment block of
+  `app/assets/javascripts/application.js`:
+
+  `//= require govuk_frontend_rails`
+
+- initialise the GOV.UK Frontend either in a .js file or in your application
   layout:
 
-`javascript window.onload = function() { window.GOVUKFrontend.initAll() };`
-- update your application markup to use the GOVUK Frontend class names
+  `window.onload = function() { window.GOVUKFrontend.initAll() };`
 
-IMPORTANT: You cannot use the GOVUK Frontend without changes if the service you
```

Comment:
What is the reason for removing this? I think it's important folks try to understand that they cannot use the kit without modification if the service it not on GOV.UK, the font for example is only licenced for GOV.UK domains?